test/e2e/docs/tests_ci.md

```diff
@@ -21,36 +22,49 @@
 
 ## Feature branch
 
-Anytime a new branch is pushed to GitHub it also becomes available in TeamCity. Build are automatically triggered for every commit for the following build configurations:
+Anytime a new branch is pushed to GitHub it also becomes available in TeamCity.
 
-- Unit tests
-- E2E Tests (mobile)
-- E2E Tests (desktop)
-- Code style
+| Build configuration name | Automatically triggered? |
+| ------------------------ | ------------------------ |
+| E2E Tests (mobile)       | Yes                      |
+| E2E Tests (desktop)      | Yes                      |
+| Pre-Release Tests        | No                       |
 
-Some build configurations are not triggered automatically, but are available to be run if manually triggered:
-
-- Quarantined E2E
-- Pre-Release E2E
-
-![teamcity_branch_view](resources/teamcity_branch_view.png)
+![teamcity_branch_view](./resources/teamcity_feature_branch_animation.gif)
```

Comment:
Do you think we should consider these gifs sensitive and only include them on the FG? I'm wondering because our TC servers are behind the proxy and not publicly available. I don't see anything sensitive in the gifs, but I wonder if we might want to double check with systems or someone else before including gifs of TC here.

Reply:
It's a good point and I can see it going both ways.

There indeed isn't any information that are sensitive in the gifs, and we have already had the screenshots as part of documentation for some time - so the ship has already half-sailed if this was a problem. I was also careful to not capture the URL schema of our TeamCity instance.

If we move these behind into the FG it would be preferable to have a link to the page, but that is likely not allowed - we have to beat around the bush to even mention the FG, for example. Ultimately I don't think we have a good answer here and any paths we take will have unintended consequences.

Reply:
Removed in e509019654ce00c87401cae9c814dcba2774a582.